src/main.css

```diff
@@ -68,10 +65,24 @@ form {
 
 form input {
   margin: 10px 0;
-  padding: 5px;
+  padding: 8px 5px;
+}
+
+button {
+  background: #433a94;
+  color: #dfdbdb;
+  border: none;
+  padding: 10px;
+  transition: all 0.5s ease-in-out;
+  box-shadow: 3px 3px 5px #100357;
+  margin: 5px 10px;
+}
+
+button:hover {
+  background: #1e21ca;
+  transform: scale(1.05);
```

Comment:
- [ ] Kindly update the Getting started section of your README file, it currently carries a link to the To-Do List project instead of the Leaderboard project. This might confuse anyone that wants to clone your app.